Libs/MRML/Core/vtkMRMLNode.h

```diff
@@ -152,6 +178,10 @@ class VTK_MRML_EXPORT vtkMRMLNode : public vtkObject
   /// \note Subclasses should implement this method
   virtual vtkMRMLNode* CreateNodeInstance() = 0;
 
+  /// \brief Returns true if the class supports deep and shallow copying node content
+  /// (implements ShallowCopyContent and DeepCopyContent).
```

Comment:
`(implements ShallowCopyContent and DeepCopyContent).` -> `(implements CopyContent(vtkMRMLNode* anode, bool deepCopy/*=true*/)).`